src/app/components/AddEvaluationForm/AddEvaluationForm.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState } from 'react';
+import { Rating } from 'react-simple-star-rating';
+import styled from 'styled-components';
+import useEvaluations from '../../hooks/useEvaluations';
+import type { Pupil } from '../../types/types';
+import Button from '../Button/Button';
+import Navigation from '../Navigation/Navigation';
+
+type AddEvaluationFormProps = {
+  pupil: Pupil;
+  onSubmit: (category: string, evaluation: string) => void;
+  onCancel: () => void;
+  missingInput: boolean;
+};
+
+export default function AddEvaluationForm({
+  pupil,
+  onSubmit,
+  onCancel,
+}: AddEvaluationFormProps): JSX.Element {
+  const { evaluations } = useEvaluations();
+  const [selectedCategory, setSelectedCategory] = useState('');
+  const [selectedEvaluation, setSelectedEvaluation] = useState('');
+  const [rating, setRating] = useState(0);
```

Comment:
looks a bit inconsistent, maybe remove selected from the first two states or rename rating to selectedRating